app/views/devise/registrations/edit.html.erb

```diff
@@ -38,7 +38,7 @@
     <small class="text-sm block text-right">
       <p><%= t(".unhappy") %> <%= link_to t(".cancel_my_account"), registration_path(resource_name), data: { confirm: t(".are_you_sure") }, method: :delete %></p>
 
-      <%= link_to t("devise.shared.links.back"), :back %>
+      <%= link_to t("devise.shared.links.back"), :back, class: "Link Link--info Link--bs inline-block" %>
```

Comment:
I'd assume we would want to replace all link_to with react_component "Link"

Reply:
As long as we can get along just fine using the classes of the react components, that's perfectly fine.
I would actually say it's preferable since it's easier to test in view specs for instance.

This is the reason that we are using BEM over scoped styling / styled components / etc. - it allows us to just use the component styling, which is mostly enough for buttons / inputs / alerts etc.

Reply:
However: All this "class spam" makes me wonder if it even makes sense to have a Link component in Muffibook.

All of this could very likely have just been:
```suggestion
      <%= link_to t("devise.shared.links.back"), :back, class: "text-info inline-block" %>
```

I mean, the point of the BEM-classes is to make the more elaborate styling easier to implement. If we instead make simple styling harder to implement, I think we are not doing it right.

I'll take this up with Jacob.